proposed-api-sample/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"enableProposedApi": true,
+	"name": "proposed-api-sample",
+	"displayName": "proposed-api-sample",
+	"description": "Sample showing how to use Proposed API",
+	"version": "0.0.1",
+	"publisher": "vscode-samples",
+	"repository": "https://github.com/Microsoft/vscode-extension-samples",
+	"engines": {
+		"vscode": "^1.34.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onCommand:extension.helloWorld"
+	],
+	"main": "./out/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "extension.helloWorld",
+				"title": "Hello World"
+			}
+		]
+	},
+	"scripts": {
+		"vscode:prepublish": "npm run compile",
+		"compile": "tsc -p ./",
+		"lint": "tslint -p ./",
+		"watch": "tsc -watch -p ./",
+		"postinstall": "vscode-dts dev"
```

Comment:
Right now and for the remainder of this iteration this will be broken because extension terminals were stabilized and proposed still depends on stable for that and TerminalDimensions (https://github.com/microsoft/vscode/issues/80428):

```
~/dev/Microsoft/vscode-extension-samples/proposed-api-sample @pine/proposed-api-sample │ ?2
❯ yarn compile
yarn run v1.13.0
warning package.json: No license field
$ tsc -p ./
vscode.proposed.d.ts:717:24 - error TS2304: Cannot find name 'TerminalDimensions'.

717     readonly dimensions: TerminalDimensions;
                             ~~~~~~~~~~~~~~~~~~

vscode.proposed.d.ts:751:24 - error TS2304: Cannot find name 'TerminalDimensions'.

751     readonly dimensions: TerminalDimensions | undefined;
                             ~~~~~~~~~~~~~~~~~~

vscode.proposed.d.ts:852:53 - error TS2304: Cannot find name 'Pseudoterminal'.

852     constructor(callback: (thisArg?: any) => Thenable<Pseudoterminal>);
                                                          ~~~~~~~~~~~~~~

vscode.proposed.d.ts:859:41 - error TS2304: Cannot find name 'Pseudoterminal'.

859     callback: (thisArg?: any) => Thenable<Pseudoterminal>;
                                              ~~~~~~~~~~~~~~


Found 4 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I think the sample should workaround this issue by depending on the master vscode.d.ts as well a vscode.proposed.d.ts. We should remove @types/vscode and update scripts to something like this:

```
"downloadapi": "vscode-dts master",
"postdownloadapi": "vscode-dts master",
"postinstall": "npm run downloadapi"
```

Reply:
👍 Yeah, I would say being on latest vscode.d.ts when using proposed API is good recommendation.

Reply:
Updated `postinstall` script and added a note in docs: https://github.com/microsoft/vscode-docs/commit/681f9ef03144321eb7b3bfc3171a60304e89e6b6